addons/point_of_sale/models/account_move.py

```diff
@@ -10,6 +10,7 @@ class AccountMove(models.Model):
     pos_order_ids = fields.One2many('pos.order', 'account_move')
     pos_payment_ids = fields.One2many('pos.payment', 'account_move_id')
     pos_refunded_invoice_ids = fields.Many2many('account.move', 'refunded_invoices', 'refund_account_move', 'original_account_move')
+    reversed_pos_order_id = fields.Many2one('pos.order', string="Reversed POS Order")
```

Comment:
"reversed_pos_order_id" is not entirely clear what it is about.  

Maybe with an explanation in help="The pos order that was reverted after closing the session to create an invoice for it. "

Or with a better name, but it quite complicated...  

Otherwise, we can  continue with this.

Reply:
@jco-odoo, I applied the suggested changes.